src/core/recv_buffer.c

```diff
@@ -186,6 +255,7 @@ QuicRecvBufferResize(
     _In_ uint32_t TargetBufferLength
     )
 {
+    CXPLAT_DBG_ASSERTMSG(RecvBuffer->RecvMode != QUIC_RECV_BUF_MODE_EXTERNAL, "Should never resize in External mode");
```

Comment:
Generally, we try to keep code lines to <= 80 characters. Comments can go to 120.
```suggestion
    CXPLAT_DBG_ASSERTMSG(
        RecvBuffer->RecvMode != QUIC_RECV_BUF_MODE_EXTERNAL,
        "Should never resize in External mode");
```